kiwi/builder/disk.py

```diff
@@ -1125,7 +1125,8 @@ def _write_generic_fstab(
             custom_root_mount_args += ['ro']
             fs_check_interval = '0 0'
 
-        if self.volume_manager_name and self.volume_manager_name == 'btrfs':
+        if self.volume_manager_name and self.volume_manager_name == 'btrfs' \
+           and self.btrfs_set_default_volume is False:
```

Comment:
We need to handle the unset case too.

Reply:
why ? None is handled as True -> default volume is wanted and set and should not appear in fstab